aw/Extensions/Reach.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import MapKit
+
+extension Reach {
+
+    var readingFormatted: String {
+        guard let lastGageReading = lastGageReading, let unit = unit else {
+            //print(self.lastGageReading, self.unit)
+            return "n/a"
+        }
+        return "\(lastGageReading) \(unit)"
+    }
+
+    var color: UIColor {
+        let con = Condition.fromApi(condition: condition ?? "")
+
+        return con.color
+    }
+
+    var icon: UIImage? {
+        return Condition.fromApi(condition: condition ?? "").icon
+    }
+
+    var sectionCleanedHTML: String? {
+        if let section = section,
+            let data = section.data(using: .utf8),
+            let html = try? NSMutableAttributedString(
+                data: data,
+                options: [NSAttributedString.DocumentReadingOptionKey.documentType: NSAttributedString.DocumentType.html],
+                documentAttributes: nil) {
+            return html.string
+        } else {
+            return nil
+        }
+    }
+
+    var distanceFormatted: String? {
+        return distance != 0 ? "\(Int(distance)) mi" : ""
+    }
+
+    var updatedString: String? {
```

Comment:
'UpdateAt' will imply that this is a time rather than a string that was updated.